ArtOfIllusion/src/artofillusion/procedural/ProcedureOwner.java

```diff
@@ -22,20 +24,14 @@ define the user interface for editing it. */
   
   /** Create an object which displays a preview of the procedure. */
   
-  default Object getPreview(ProcedureEditor editor)
+  default MaterialPreviewer getPreview()
```

Comment:
Changing the signature here technically breaks compatibility at both a source and binary level. While updating the core plugins is simple, have you tested that this does not break TaPD, which builds of the procedural system?